kategory-docs/docs/_includes/_main.html

```diff
@@ -1,22 +1,22 @@
 <main id="content" class="indirect-injection-final">
     <section class="features">
         <div class="container">
-            <a href="#" class="feature-item">
+            <a href="/docs/typeclasses/functor/" class="feature-item">
                 <img class="feature-img" src="img/typeclasses_icon.svg" alt="">
                 <h2 class="feature-title">Typeclasses</h2>
                 <p>Functor, Applicative, Monad and all their friends so you can take advantage of ad-hoc polymorphism in Kotlin</p>
             </a>
-            <a href="#" class="feature-item">
+            <a href="/docs/datatypes/option/" class="feature-item">
                 <img class="feature-img" src="img/datatypes_icon.svg" alt="">
                 <h2 class="feature-title">Data types</h2>
                 <p>Option, Try, Either, Eval, NonEmptyList and many other data types based on algebraic properties</p>
             </a>
-            <a href="#" class="feature-item">
+            <a href="/docs/patterns/monad_comprehensions/" class="feature-item">
```

Comment:
For consistency I'd point it to the Error Handling entry, as it's the one we're using inside the docs as entry point.